hrcase/CRM/HRCase/Upgrader.php

```diff
@@ -190,7 +190,13 @@ public function upgrade_1431() {
    * @return bool
    */
   public function upgrade_1432() {
-    $result = civicrm_api3('CustomField', 'get', ['name' => 'Category']);
+    $result = civicrm_api3('CustomField', 'get', [
+      'name' => 'Category',
+      'api.OptionGroup.get' => [
+        'id' => '$value.option_group_id',
+        'name' => 'case_type_category'
+      ],
+    ]);
```

Comment:
@ajesamson sorry for the confusion, on my last comment I meant the custom group id instead of the option group. Using the option group id like you did here will not help getting the right custom field.

So basically, what you need to do here is something like:

```php
civicrm_api3('CustomField', 'get', [
  'name' => 'Category',
  'custom_group_id' => 'case_type_category'
]);
```